extensions/amp-form/0.1/amp-form.js

```diff
@@ -72,13 +79,27 @@ export class AmpForm {
     }
     if (this.xhrAction_) {
       e.preventDefault();
+      this.setState_(FormState_.SUBMITTING);
```

Comment:
I'd only do it for `POST` and `_top` (on top of what you already have, obviously)